apps/docs/content/guides/database/extensions/pgvector.mdx

```diff
@@ -102,6 +102,19 @@ const { data, error } = await supabase.from('posts').insert({
 })
 ```
 
+## Specific usage cases
+
+### Queries with filtering
+
+If you use IVFFlat or HNSW index and filter results from the query 
+E.g. 
+```
+SELECT * FROM items WHERE category_id = 123 ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+```
+filters results of vector column embedding by another column ```category_id```.
```

Comment:
````suggestion
For example, the following query may return fewer than 5 rows, even if 5 corresponding rows exist in the database. This is because the embedding index may not return 5 rows matching the filter.

```
SELECT * FROM items WHERE category_id = 123 ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
```
````